docs/architecture/execution-engines.md

```diff
@@ -0,0 +1,42 @@
+## Execution Engines(?)
+
+A workload management system for the scheduling and running of jobs.  The typical enterprise requires support for multiple operating systems.  A core tenet of the project is to leverage both open source projects and commercial cloud products and their capabilities where we can. 
```

Comment:
I meant it as a definition like:

Execution Engine:
A workload management system for the scheduling and running of jobs.

I'll reformat.